deb-get

```diff
@@ -200,10 +200,17 @@ function upgrade_apt() {
     ${ELEVATE} apt-get -q -o Dpkg::Progress-Fancy="1" -y upgrade
 }
 
```

Comment:
Permissions  got changed  100755 → 100644
please could you change them back to 755

Reply:
Yes, it seems that whenever GitHub does a rebase from the web interface, it ignores the permissions. I opened #553 to work around those cases in the test workflow.